src/components/container/BloodPressureVisualization/BloodPressureVisualization.tsx

```diff
@@ -0,0 +1,257 @@
+import "./BloodPressureVisualization.css";
+import { useEffect, useState } from "react";
+import { debounce } from "lodash";
+import MyDataHelps, { Guid, SurveyAnswer, SurveyAnswersPage, SurveyAnswersQuery } from "@careevolution/mydatahelps-js";
+import plus from './plus.svg';
+import React from "react";
+import { Button, DateRangeNavigator, LoadingIndicator } from "../../presentational";
+import DumbbellChart from "../../presentational/DumbbellChart";
+import { Dumbbell, ClosedInterval, Axis, DataPoint, DumbbellClass } from "../../presentational/DumbbellChart/DumbbellChart";
+import { previewSurveyData } from './BloodPressureVisualization.previewdata'
+import { addDays, format, isEqual, isMonday, nextSunday, parseISO, previousMonday, startOfDay } from "date-fns";
+
+export type DataFaucet = "PreviewSurvey" | "Survey";
+
+export interface BloodPressureVisualizationProps {
+    source : DataFaucet,
+    surveyName?: string,
+    stepIdentifier? : string,
+    dateOfResultId? : string,
+    systolicResultId? : string,
+    diastolicResultId? : string
+};
+
+export interface IBloodPressureDataPoint {
+    dateLabel: string,
+    date: Date,
+    systolic : number,
+    diastolic : number
+};
+
+interface IBloodPressureDumbbell {
```

Comment:
I would not prefix interfaces with "I" - that is not something you do in typescript.  Interface should just be "BloodPressureDummbell"